docs/tutorials/ad-hoc-simulation/planning-simulation.md

```diff
@@ -2,81 +2,134 @@
 
 ## Preparation
 
-1. Download and unpack a sample map.
+Download and unpack a sample map.
 
-   - Click [here](https://drive.google.com/file/d/1499_nsbUbIeturZaDj7jhUownh5fvXHd/view?usp=sharing) to download.
-   - Unpack it by running the following command.
+- Click [here](https://drive.google.com/file/d/1499_nsbUbIeturZaDj7jhUownh5fvXHd/view?usp=sharing) to download.
+- Unpack it by running the following command.
 
-   ```bash
-   unzip -d ~/Downloads/ ~/Downloads/sample-map-planning.zip
-   ```
+```bash
+unzip -d ~/Downloads/ ~/Downloads/sample-map-planning.zip
+```
 
-### Note
+!!! Note
 
-- Sample map: Copyright 2020 TIER IV, Inc.
+    Sample map: Copyright 2020 TIER IV, Inc.
 
-## How to run a planning simulation
+## Basic Simulations
 
-1. Launch Autoware.
+### Lane Driving
 
-   ```sh
-   source ~/autoware/install/setup.bash
-   ros2 launch autoware_launch planning_simulator.launch.xml map_path:=$HOME/Downloads/sample-map-planning vehicle_model:=sample_vehicle sensor_model:=sample_sensor_kit
-   ```
+#### 1. Launch Autoware
 
-   Note that you cannot use `~` instead of `$HOME` here.
+```sh
```

Comment:
Although this was originally wrong, could you replace `sh` with `bash` to be clear and consistent?

```suggestion
```bash
```